prow/spyglass/README.md

```diff
@@ -75,7 +75,7 @@ The following lenses are available:
 deck:
   spyglass:
     size_limit: 100000000  # 100 MB
-    gcs_browser_prefix: https://gcsweb.k8s.io/gcs/
+    gcs_browser_prefix: https://gcsweb.k8s.io/gs/
```

Comment:
gcsweb also now accepts the gs prefix??

Reply:
bah. no.

Reply:
fixed, and also the line above, but not the line before that (which is plank)